src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryConfig.java

```diff
@@ -112,11 +112,6 @@
                       "the ResourceResolver mapping. The default value is /etc/map.")
     String resource_resolver_map_location() default MapEntries.DEFAULT_MAP_ROOT;
 
-    @AttributeDefinition(name = "Mapping Observation",
```

Comment:
The mapping observation configuration setting is used to register the `ResourceChangeListener`, so that the `MapEntries` will be notified of changes that it is interested in. In AEM, it is set to `/`. 

However, given the various configurations that exist, it is easy to compute the right paths to use. (That is also changed in the part where the listener is registered.)

There are also some fallbacks in there to listen to `/` anyway, if that is required.

Reply:
I think it may be confusing to remove this setting. Perhaps mark it as deprecated and indicate that it is actually auto-computed?

Reply:
Might also be worth logging a warning if a non-default setting is found.